hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/ClientIds.java

```diff
@@ -163,6 +163,7 @@ private void updateHeartbeat() throws HoodieHeartbeatException {
 
   private void updateHeartbeat(Path heartbeatFilePath) throws HoodieHeartbeatException {
     try (OutputStream outputStream = this.fs.create(heartbeatFilePath, true)) {
+      LOG.debug("create hearbeat file in path: {}", heartbeatFilePath);
     } catch (IOException io) {
       throw new HoodieHeartbeatException("Unable to generate heartbeat for file path " + heartbeatFilePath, io);
```

Comment:
Does a comment work here?

```java
// no operation
```

Reply:
yes, add the comments already @danny0405

Reply:
I mean to replace the debug log with a comment.